Maven.md

```diff
@@ -159,6 +159,46 @@ The summary looks typically like in the screenshot below.
 
 ![image](https://user-images.githubusercontent.com/231070/146644224-698f6dbe-fa1c-4632-8051-0e512226f577.png)
 
+### Performance characteristics
+
+<details>
+  <summary>Click to see more details about each of the calculated performance characteristics.</summary>
+
+#### Initial build time
+
+The elapsed time of the first build without any build performance acceleration measures explicitly applied by the experiment.
+
+#### Build time with instant savings
+
+The elapsed time of the second build with build performance acceleration measures explicitly applied by the experiment. The experienced build performance acceleration typically comes from build caching (experiments 1 to 4).
+
+Achieving the build time with instant savings does not require any changes to the goals of the build.
+
+#### Build time with pending savings
+The projected elapsed build time of the second build with build performance acceleration measures explicitly applied by the experiment. The projection assumes that all cacheability issues of the executed cacheable goals get resolved, and it takes into account the degree of parallelization in project execution.
+
+Achieving the build time with pending savings requires changes to the executed cacheable goals of the build.
+
+#### Avoided cacheable goals
+
+The estimated reduction in serial execution time of the goals of the second build due to reusing the goals outputs from the first build. The avoidance savings are calculated as the difference between the time required to reuse the goals outputs and the time it took to create the goals outputs originally.
+
+#### Executed cacheable goals
+
+The serial execution time of the goals executed in the second build that Gradle considered cacheable. These goals stored their outputs in the build cache during the first build but were unable to reuse the outputs during the second build.
```

Comment:
> that Gradle considered cacheable

This sounds a little strange in the Maven README. What do you think about this:

> that Gradle Enterprise considered cacheable

Reply:
Same below in "Executed non-cacheable goals".

Reply:
Agreed. Fixed in https://github.com/gradle/gradle-enterprise-build-validation-scripts/commit/938475e1a6cec5b541e755764f66b805e056b3af